doc/source/building/linux.rst

```diff
@@ -102,106 +54,11 @@ require administrator rights.
 
     LD_PRELOAD=/path/to/libatlas.so.3 ./my-application
 
-
 Fedora 26
-=========
+---------
 
 To install scipy build requirements, you can do::
 
     sudo dnf install gcc-gfortran python3-devel openblas-devel lapack-devel
```

Comment:
Maybe explicit is better, but I normally do something like:
```
$ sudo dnf builddep scipy
```

I think apt has something similar.

Reply:
Maybe [`build-dep`](https://linux.die.net/man/8/apt-get)?
> **build-dep** source_pkg
    Causes **apt-get** to install/remove packages in an attempt to satisfy the build dependencies for a source package.

Reply:
I forgot to mention.  One thing I like about using
```
$ sudo dnf builddep scipy
```
is you don't have to worry about updating the list of build dependencies when names change or new packages become required.

Reply:
Do you need any extra config for this to work, like a PKGBUILD file? I know [you need one for Arch](https://wiki.archlinux.org/title/Pacman/Rosetta)